org.flightgear.FlightGear.appdata.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 Richard Hughes <richard@hughsie.com> -->
+
+<!-- HOUSEKEEPING, REMOVE THIS COMMENT WHEN THIS GOES UPSTREAM
+BugReportURL: https://sourceforge.net/p/fgrun/support-requests/9/
+SentUpstream: 2014-09-17
+-->
+
+<application>
+  <id type="desktop">org.flightgear.FlightGear</id>
```

Comment:
Seems to be an odd mix between old and new format. Run `appstream-util upgrade` on it perhaps.

Reply:
Will do